pkg/secret/aes.go

```
@@ -0,0 +1,40 @@
+	"crypto/cipher"
+	"crypto/rand"
+	"io"
+)
+
+func EncryptAES(plaintext string, key []byte) ([]byte, []byte, error) {
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return nil, nil, err
+	}
+	nonce := make([]byte, 12)
+	io.ReadFull(rand.Reader, nonce)
```

golangci-lint (1.64.8)

Error return value of `io.ReadFull` is not checked

(errcheck)